buildbot-conf/cordova.conf

```diff
@@ -360,9 +360,10 @@ def cordova_steps_run_tests(platform, extra_args=list()):
                 'log',
                 '--platform', platform
             ],
-            description   = 'gathering logs',
-            timeout       = LOG_GETTING_TIMEOUT,
-            haltOnFailure = False,
+            description    = 'gathering logs',
+            timeout        = LOG_GETTING_TIMEOUT,
+            haltOnFailure  = False,
+            flunkOnFailure = False,
```

Comment:
Good call with this one! However, later on we should also debug what's causing this error on Windows: `Couldn't find log file. It may be because of old cordova-windows version or some problems with log script.`.